runtime/libpgmath/lib/x86_64/libm_amd.h

```diff
@@ -89,6 +89,8 @@ typedef unsigned long long __UINT8_T;
 # define TLOSS          5
 # define PLOSS          6
 
+/* The BSDs (and probably macOS) define HUGE in math.h.  */
```

Comment:
Also I've confirmed that it is the case that `HUGE` is defined in `math.h` on macOS.